.github/workflows/push.yml

```diff
@@ -0,0 +1,40 @@
+name: Check build
+
+on:
+  push:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  builddebug:
+    name: Debug
+    uses: actions/checkout@v3
+    with:
+      jobname: Build
+      unittests: true
+      cmakeopts: >
+        -DCMAKE_BUILD_TYPE=Debug -DKPSR_WITH_DOXYGEN=true
+        -DKPSR_TEST_PERFORMANCE=true -DKPSR_WITH_SOCKET=true -DKPSR_WITH_CODE_METRICS=true
+
+  buildrelease:
+    needs: builddebug
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Vanilla
+            zmq: "false"
+          - name: ZMQ
+            zmq: "true"
```

Comment:
I don't think we need a matrix here because zmq and non-zmq versions of the sdk are the same.

Reply:
@mharshe note that this conf comes from kpe-sdk, should I update there too?

Reply:
> I don't think we need a matrix here because zmq and non-zmq versions of the sdk are the same.

Do you mean remove `zmq:` entries and previous `-D`?

Reply:
Maybe for another task later. I think I added this thinking that core-zmq will search for sdk-zmq and it should not fail but from what I have seen in the CI, it makes the correct decision and both zmq/non-zmq will find sdk correctly.

Reply:
Noted.

Reply:
No -- there is no kpe-sdk-zmq package (and there shouldn't be either). So remove the zmq entries here, and in a later task, remove zmq entries from kpe-sdk.